policyengine_us/variables/gov/states/tx/twc/ccs/copay/tx_ccs_copay_waived.py

```
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
```

Ruff (0.16.1)

[error] `from policyengine_us.model_api import *` used; unable to detect undefined names

(F403)